packages/rulesets/src/spectral/functions/patch-response-codes.ts

```diff
@@ -44,6 +46,13 @@ export const patchResponseCodes = (patchOp: any, _opts: any, ctx: any) => {
         path: path,
       })
     }
+
+    if (patchOp.responses["202"]?.schema) {
+      errors.push({
+        message: "202 response for a LRO PATCH operation must not have a response schema specified.",
+        path: path,
```

Comment:
pl update the documentation as well to call out this condition.

Reply:
Updated